include/dlaf/lapack/tile.h

```diff
@@ -604,20 +721,49 @@ internal::CusolverInfo<T> potrfInfo(cusolverDnHandle_t handle, const blas::Uplo
 template <class T>
 void potrf(cusolverDnHandle_t handle, const blas::Uplo uplo, const matrix::Tile<T, Device::GPU>& a) {
   auto info = potrfInfo(handle, uplo, a);
-  assertExtendInfo(dlaf::gpulapack::internal::assertInfoPotrf, handle, std::move(info));
+  assertExtendInfo(dlaf::gpulapack::internal::assert_info_potrf, handle, std::move(info));
 }
 
+template <class T>
+internal::CusolverInfo<T> trtri_info(cusolverDnHandle_t handle, const blas::Uplo uplo,
+                                     const blas::Diag diag, const matrix::Tile<T, Device::GPU>& a) {
+  DLAF_ASSERT(square_size(a), a);
+  const auto n = a.size().rows();
+
+  internal::CusolverInfo<T> info{};
+#ifdef DLAF_WITH_CUDA
+  DLAF_STATIC_UNIMPLEMENTED(T);
```

Comment:
Is it because of additional workspace needed by the cusolver API?

Reply:
cusolver doesn't supply a trtri function

Reply:
https://docs.nvidia.com/cuda/cusolver/index.html?highlight=cusolverDnXtrtri#cusolverdnxtrtri isn't this the one we need?

Reply:
We use the legacy API which doesn't provide it...
https://docs.nvidia.com/cuda/cusolver/index.html?highlight=cusolverDnXtrtri#dense-linear-solver-reference-legacy
API update better be done in a different PR.